### PR TITLE
Feature/create render info on header

### PR DESCRIPTION
**Pull request by DKnehi**

### CODE REVIEW

File: javascript-practice/src/js/views/userView.js [hunk 1]
```diff
@@ -17,9 +17,12 @@ export default class UserView {
     this.passwordErrorEl = document.getElementById('password-error');
     this.arrowEl = document.getElementById('arrow');
     this.logoutEl = document.getElementById('logout');
+    this.firstNameEl = document.getElementById('firstName');
+    this.headerNameEl = document.getElementById('headerName');
+    this.lastNameEl = document.getElementById('lastName');
+    this.roleEl = document.getElementById('role');
```
Comment (hoangle-agilityio) [inline javascript-practice/src/js/views/userView.js:23]: Pls arrange to name follow alpha b. From line 13 to 23 with the same format

File: javascript-practice/src/js/views/userView.js [hunk 2]
```diff
@@ -73,7 +76,26 @@ export default class UserView {
...
+  };
+
+
   redirectPage = (page) => {
     window.location.replace(page);
   };
 }
+
+//I will research and edit this later
+const a = new UserView();
```
Comment (hoangle-agilityio) [inline javascript-practice/src/js/views/userView.js]: naming?

File: javascript-practice/src/pages/dashboard.html [hunk 3]
```diff
@@ -23,16 +22,16 @@ <h2 class="header-dashboard-name">User</h2>
           <div class="header-dashboard-profile-box flex">
             <div class="header-dashboard-avatar"></div>
             <div>
-              <!-- TODO: this is fake data. I'll update it in the next task -->
               <div class="flex header-dashboard-profile-name-box">
-                <h2 class="header-dashboard-profile-name">FirstName</h2>
-                <h2 class="header-dashboard-profile-name">LastName</h2>
+                <h2 id="firstName" class="header-dashboard-profile-name"></h2>
```
Comment (hoangle-agilityio) [inline javascript-practice/src/pages/dashboard.html]: I think you just need only one element `h2` for both the First and Last Name. And we will format the name in JS and append it here

File: javascript-practice/src/js/controllers/userController.js [hunk 4]
```diff
@@ -21,6 +21,9 @@ export default class UserController {
 
       if (userData) {
         this.view.redirectPage('dashboard.html');
+        localStorage.setItem('firstName', userData.firstName);
```
Comment (hoangle-agilityio) [inline javascript-practice/src/js/controllers/userController.js]: Pls create constant for key of local storage. Ex: `FIRST_NAME: "firstName"`

Comment (DKnehi) [reply javascript-practice/src/js/controllers/userController.js]: i done

File: javascript-practice/src/js/views/userView.js (hunk 1 above)
Comment (DKnehi) [reply javascript-practice/src/js/views/userView.js:23]: i done

File: javascript-practice/src/pages/dashboard.html (hunk 3 above)
Comment (DKnehi) [reply javascript-practice/src/pages/dashboard.html]: i done

File: javascript-practice/src/js/views/userView.js [hunk 5]
```diff
@@ -73,7 +75,22 @@ export default class UserView {
...
+    }
+  };
+
   redirectPage = (page) => {
     window.location.replace(page);
   };
 }
+
+const renderUserInfo = new UserView();
+renderUserInfo.showUserInfo()
```
Comment (hoangle-agilityio) [inline javascript-practice/src/js/views/userView.js]: Did you use a prettier format? I don't see an empty end line.

File: javascript-practice/src/js/controllers/userController.js [hunk 6]
```diff
@@ -21,6 +21,9 @@ export default class UserController {
 
       if (userData) {
         this.view.redirectPage('dashboard.html');
+        localStorage.setItem('FIRST_NAME', userData.firstName);
```
Comment (hoangle-agilityio) [inline javascript-practice/src/js/controllers/userController.js]: Pls re-check. My idea is create constant instead of use directly

**Pull request merged**